### PR TITLE
connect

### PR DESCRIPTION
**Pull request by Y4kuzi**
connect class

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,4 +1,17 @@
 print('Sup world!')
```
Comment (malvinarios) [inline main.py:1]: Leave for debugging?

Comment (Y4kuzi) [reply main.py:1]: I just removed the print shit and did a commit

File: main.py [hunk 2]
```diff
@@ -1,4 +1,17 @@
...
-for _ in range 10:
-  print(f"Go nuts {_} time(s)")
+import socket
+
+class Bot:
+    def __init__(self):
+        self.server = "irc.provisionweb.org"
+        self.port = 6667
+        self.nick = "ShitCodedBot"
+        self.sock = socket.socket()
```
Comment (malvinarios) [inline main.py]: Ident should't be required?

**Comment by Y4kuzi** [comment]
fixed your loop

File: main.py (hunk 2 above)
Comment (Y4kuzi) [reply main.py]: Well, yeah eventually but we haven't made anything beyond the connect. We can do that now

**Review by malvinarios — APPROVED** [review]

File: main.py (hunk 2 above)
Comment (malvinarios) [reply main.py]: I thought it's required to even connect to irc server

**Comment by Y4kuzi** [comment]
Sending data to the server

**Pull request merged**